src/Centreon/Application/Controller/Monitoring/MetricController.php

```diff
@@ -113,7 +113,7 @@ private function extractDatesFromRequestParameters(RequestParametersInterface $r
         $start = new \DateTime($start);
         $end = new \DateTime($end);
 
-        if ($start > $end) {
+        if ($start >= $end) {
             throw new \RangeException('End date must be greater than start date.');
```

Comment:
```suggestion
            throw new \RangeException('End date must be greater than or equal to start date.');
```

Reply:
Since I've committed @sc979 's suggestion, end date need to be strictly greater ;)

Reply:
Are you sure? Cause for me the commit is 
from      >        to        >=

Reply:
if start date is > or = end date, then an error is throwed. So end date must be greater

Reply:
Ah yes! Sorry, I hadn't noticed that the error was displayed on "end Date" and not "start Date".